BSNSS-38: Update README (#55)
ovotech/bit-node-tools: README.md

| [@ovotech/bigquery-pg-sink](packages/bigquery-pg-sink/README.md)               | Used in prod | Stream the results of query made by [nodejs-bigquery](https://github.com/googleapis/nodejs-bigquery) into a [postgres database](https://www.postgresql.org/).             |
| [@ovotech/influx-metrics-tracker](packages/influx-metrics-tracker/README.md)   | Used in prod | Track metrics and store them in an Influx database, with secondary [logging](packages/winston-logger/README.md) if Influx is unavailable.                                 |

## Getting Started

These instructions will get you a copy of the project up and running on your local machine for development and testing purposes.

### Prerequisites

To get up and running, you will need the following tools. Use [Homebrew](https://docs.brew.sh/Installation) for installation where possible.

#### Code

> To write and test code you will need [node v12.4.1+](https://nodejs.org/en/) and [yarn](https://yarnpkg.com/lang/en/) installed. It has been tested with node version 12.4.1.

Reviewer: ```suggestion
To install dependencies and run the code you will need [node v12.4.1+](https://nodejs.org/en/) and [yarn](https://yarnpkg.com/lang/en/) installed. It has been tested with node version 12.4.1.
```

yarn test
```

### Coding Style Tests

Code style is enforced by using a linter ([tslint](https://palantir.github.io/tslint/)) and [Prettier](https://prettier.io/).

```
yarn lint
```

## Deployment

Deployment is performed by lerna automatically on merge / push to master. You'll need to bump the package version numbers yourself. Only updated packages with newer versions will be pushed to the npm registry.

## Built With

### Languages / Core Tools

- [Typescript](http://www.typescriptlang.org/) - The primary language
- [NPM](https://www.npmjs.com/) - Node package registry

### Secondary Tooling

> - [jest](https://jestjs.io/) - Unit and integration testing framework

Reviewer 2: ```suggestion
- [jest](https://jestjs.io/) - Testing framework used for unit and integration tests
```

- [yarn](https://yarnpkg.com/lang/en/) - Typescript package management
- [lerna](https://www.npmjs.com/package/lerna) - Tool for managing JavaScript projects with multiple packages

> - [docker](https://www.docker.com/) - Deployable containers for code.

Reviewer 2: ```suggestion
- [docker](https://www.docker.com/) - Deployable containers for code
```
